### DISCUSSION TITLE
No Code configurator - Add meaningful names to items in a collection

### DISCUSSION THREAD

**Discussion by fabioS24** [mia-platform/community#344] [Ideas]
### Which area of this project could be most improved?

Backoffice

### Description

Hi, using the no code configurator it is difficult to identify a specific element within a list of elements when you have many elements. The "Item n" label is not very significant in identifying the specific item. It would be useful to be able to modify the content of this label with information from the element to make its identification easier.

![image](https://github.com/mia-platform/community/assets/93452841/8dfbb002-5ec7-403c-92a8-35d47850e6f6)

**Comment by epessina** [comment]
Hi @fabioS24 ! A new and improved tabular visualization for array of objects properties is available in [version 12.0.0](https://docs.mia-platform.eu/docs/preview/release-notes/v12.0.0#added-support-for-table-visualization-of-components-properties-of-type-array) of the Console.
